vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -537,6 +751,91 @@ float ConnectionBoxMapLookahead::get_expected_cost(
     }
 }
 
+static vtr::Rect<int> bounding_box_for_node(const t_rr_node& node) {
+    return vtr::Rect<int>(node.xlow(), node.ylow(),
+                          node.xhigh() + 1, node.yhigh() + 1);
+}
+
+static vtr::Point<int> point_for_node(const t_rr_node& node) {
```

Comment:
Function comment, and a description of why you choose this methodology.  In the case of the 7-series, it doesn't matter a lot because xhigh == xlow most of the time, but if we upstream this stuff it would be good to have a reason.

Reply:
Done.